File: app/src/main/java/com/kipucode/data/local/dao/SyncQueueDao.java
package com.kipucode.data.local.dao;

import androidx.room.Dao;
import androidx.room.Delete;
import androidx.room.Insert;

import com.kipucode.data.local.model.SyncQueue;

@Dao
public interface SyncQueueDao {
    @Insert
    void insert(SyncQueue syncQueue);
    @Delete
    void delete(SyncQueue syncQueue);
}

File: app/src/main/java/com/kipucode/data/local/dao/SyncQueueDao.kt
package com.kipucode.data.local.dao

import androidx.room.Dao
import androidx.room.Delete
import androidx.room.Insert
import com.kipucode.data.local.model.SyncQueueEntity

@Dao
interface SyncQueueDao {
    @Insert
    fun insert(syncQueue: SyncQueueEntity)

    @Delete
    fun delete(syncQueue: SyncQueueEntity)
}
